Cargo.toml

```diff
@@ -566,3 +573,18 @@ opt-level = 's'
 inherits = "release"
 codegen-units = 1
 lto = true
+
+[patch.crates-io]
+wit-bindgen = { git = "https://github.com/bytecodealliance/wit-bindgen.git" }
+wit-bindgen-rust-macro = { git = "https://github.com/bytecodealliance/wit-bindgen.git" }
+wasmparser = { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wat =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wast =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wasmprinter =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wasm-encoder =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wasm-smith =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wasm-mutate =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wit-parser =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wit-component =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wasm-wave =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
+wasm-metadata =  { git = "https://github.com/jsturtevant/wasm-tools.git", branch="unstable-refrence-stable" }
```

Comment:
this branch contains the temp fix for https://github.com/bytecodealliance/wasm-tools/issues/1995

Reply:
fixed in https://github.com/bytecodealliance/wasm-tools/releases/tag/v1.227.0